apps/discovery-platform/src/db/index.ts

```diff
@@ -193,48 +220,42 @@ export const getQuota = async (
   return dbRes;
 };
 
-export const getSumOfQuotasPaidByClient = async (
+export const getClientQuotas = async (
   dbInstance: DBInstance,
   clientId: string
-): Promise<bigint> => {
-  const text = `SELECT SUM(quota) FROM ${TABLES.QUOTAS} WHERE paid_by=$<clientId>`;
-  const values = {
-    clientId,
-  };
-  const dbRes = await dbInstance.one(text, values);
-
-  return dbRes?.sum ? BigInt(dbRes?.sum) : BigInt(0);
-};
-
-export const getSumOfQuotasUsedByClient = async (
-  dbInstance: DBInstance,
-  clientId: string
-): Promise<bigint> => {
-  const text = `SELECT SUM(quota) FROM ${TABLES.QUOTAS} WHERE client_id=$<clientId>`;
-  const values = {
-    clientId,
-  };
-  const dbRes = await dbInstance.one(text, values);
-
-  return dbRes?.sum ? BigInt(dbRes?.sum) : BigInt(0);
-};
-export const updateQuota = async (
-  dbInstance: DBInstance,
-  quota: QuotaDB
-): Promise<QuotaDB> => {
-  const text = `UPDATE ${TABLES.QUOTAS}
-  SET client_id = $<client_id>, paid_by = $<paid_by>, quota = $<quota>, action_taker = $<action_taker>
-  WHERE id = $<id>
-  RETURNING *`;
-  const values: Omit<QuotaDB, "created_at" | "updated_at"> = {
-    id: quota.id,
-    client_id: quota.client_id,
-    action_taker: quota.action_taker,
-    quota: quota.quota,
-    paid_by: quota.paid_by,
-  };
-  const dbRes: QuotaDB = await dbInstance.one(text, values);
-  return dbRes;
+): Promise<{
+  quotaPaid: bigint;
+  quotaUsed: bigint;
+  sum: bigint;
+}> => {
+  try {
+    const dbRes: {
+      payment: string;
+      quota_paid: string;
+      quota_used: string;
+    } = await dbInstance.one(
+      `SELECT payment, quota_paid, quota_used FROM ${TABLES.CLIENTS} WHERE id=$<clientId>`,
+      {
+        clientId,
+      }
+    );
+
+    const quotaPaid = BigInt(dbRes.quota_paid || 0);
+    const quotaUsed = BigInt(dbRes.quota_used || 0);
+    const sum = quotaPaid - quotaUsed;
+
+    return {
+      quotaPaid,
+      quotaUsed,
+      sum,
+    };
+  } catch {
```

Comment:
whats the error case? shouldn't we at least log something here? 
Let the call fail, because we would not want the default data when db connection is missing.